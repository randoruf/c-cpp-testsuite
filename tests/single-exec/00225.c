// Quick sort in C
#include <stdio.h>

// main function
int main() {

}