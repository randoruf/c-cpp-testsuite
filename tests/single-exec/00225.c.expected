9  3  83  9  2  0  1  65  2  822  9  11  22  3  3  3  47  
0  1  2  2  3  3  3  3  9  9  9  11  22  47  65  83  822  
